src/advanced_inv.cpp

```diff
@@ -355,7 +355,7 @@ void advanced_inventory::print_items( advanced_inventory_pane &pane, bool active
         bool stolen = false;
         if( it.has_owner() ) {
             const faction_id item_fac = it.get_owner();
-            if( item_fac != g->u.get_faction()->id ) {
+            if( it.not_owned_by_player() ) {
```

Comment:
shouldn't this take `g->u` ? or `not_owned_by_player()` really `not_owned_by_avatar()` ?

Reply:
Yep removed "not_owned_by_player()" and just use `!is_owned_by( g->u )`